tutorials/Tutorial5_Evaluation.py

```diff
@@ -1,10 +1,20 @@
-from haystack.document_stores import ElasticsearchDocumentStore
-from haystack.nodes import BM25Retriever, DensePassageRetriever, EmbeddingRetriever, FARMReader, PreProcessor
+import logging
+import tempfile
+from pathlib import Path
+
+from haystack.document_stores import ElasticsearchDocumentStore, InMemoryDocumentStore
+from haystack.pipelines import Pipeline, ExtractiveQAPipeline, DocumentSearchPipeline
+from haystack.nodes import (
+    BM25Retriever,
+    DensePassageRetriever,
+    EmbeddingRetriever,
+    FARMReader,
+    PreProcessor,
+    TextConverter,
+)
```

Comment:
For some odd reason, having the imports scattered around in Tut5 was causing import errors. Grouping the imports solved the issue.